File: Aula 04 - Estruturas de Decisões/cpp_logic/tarefaFinal.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {

    string nome_do_produto;
    int quantidade_em_estoque;
    double preco_unitario;

    // Entrada de dados
    cout << "Digite o nome do produto: "; << cin, nome_do_produto;

    cout << "Digite a quantidade em estoque: ";
    cin >> quantidade_em_estoque;

    cout << "Digite o preco unitario: ";
    cin >> preco_unitario;

    if (quantidade_em_estoque < 0) {
        cout << "Erro: A quantidade nao pode ser um valor negativo. Por favor, tente novamente." << endl;
    } else {
        cout <<"\nNome: " << nome_do_produto << endl;
        cout << "Quantidade em estoque: " << quantidade_em_estoque << endl;
        cout << "Preco unitario: R$ " << preco_unitario << endl;
    }

    return 0;
}

File: Aula 04 - Estruturas de Decisões/java_logic/tarefaFinal.java
import java.util.Scanner;

public class estoque {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String nome_do_produto;
        int quantidade_em_estoque;
        double preco_unitario;

        System.out.print("Digite o nome do produto: ");
        nome_do_produto = scanner.nextLine();

        System.out.print("Digite a quantidade em estoque: ");
        quantidade_em_estoque = scanner.nextInt();

        System.out.print("Digite o preço unitário: ");
        preco_unitario = scanner.nextDouble();

        if (quantidade_em_estoque < 0) {
            System.out.println("Erro: A quantidade não pode ser um valor negativo. Por favor, tente novamente.");
        } else {
            System.out.println("\nNome: ", nome_do_produto);
            System.out.println("Quantidade em estoque: ", quantidade_em_estoque);
            System.out.println("Preço unitário: R$ ", preco_unitario);
        }

        scanner.close();
    }
}